.github/workflows/ci.yml

```diff
@@ -0,0 +1,112 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - 'v*'
+  pull_request: {}
+  schedule:
+    - cron:  '0 3 * * *' # daily, at 3am
+
+jobs:
+  #lint:
+  #  name: Linting
+  #  runs-on: ubuntu-latest
+  #
+  #  steps:
+  #    - uses: actions/checkout@v1
+  #    - uses: actions/setup-node@v1
+  #      with:
+  #        node-version: 10.x
+  #
+  #    - name: get yarn cache dir
+  #      id: yarn-cache
+  #      run: echo "::set-output name=dir::$(yarn cache dir)"
+  #
+  #    - uses: actions/cache@v1
+  #      with:
+  #        path: ${{ steps.yarn-cache.outputs.dir }}
+  #        key: ${{ runner.os }}-yarn-lint-${{ hashFiles('**/yarn.lock') }}
+  #        restore-keys: |
+  #          ${{ runner.os }}-yarn-
+  #
+  #    - name: install dependencies
+  #      run: yarn install
+  #
+  #    - name: lint
+  #      run: yarn lint
+
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        test-suite:
+          - test:one ember-1.13
+          - test:one ember-lts-2.4
+          - test:one ember-lts-2.8
+          - test:one ember-3.0
+          - test:one ember-3.0
+          - test:one ember-lts-3.4
+          - test:one ember-lts-3.8
+          - test:one ember-lts-3.12
+          - test:one ember-release
+          - test:one ember-beta
+          - test:one ember-canary
+        include:
+          - test-suite: "test:one ember-canary"
+            allow-failure: true
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+
+      - name: get yarn cache dir
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v1
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-test-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: install dependencies
+        run: yarn install
+
+      - name: tests
+        run: yarn ${{ matrix.test-suite }}
```

Comment:
why not put the `test:one` in here instead? 🤔

Reply:
👍